host-configs/TOTAL/pangea3-gcc8.4.1-openmpi-4.1.2.cmake

```diff
@@ -44,7 +43,7 @@ if (DEFINED ENV{MPI_ROOT})
   set(MPIEXEC                $ENV{MPI_ROOT}/bin/mpirun  CACHE STRING "")
   set(ENABLE_WRAP_ALL_TESTS_WITH_MPIEXEC ON CACHE BOOL "")
 else()
-  message(FATAL_ERROR "You must have MPI_ROOT variable set, we advise loading module smpi/10.3.0.1")
+  message(FATAL_ERROR "You must have MPI_ROOT variable set, we advise loading module openmpi/p3/4.1.2")
```

Comment:
If you want a specific version, maybe can you check that `MPI_ROOT` has the exact proper value? (up to you, of course.)

Reply:
That message is not correct this module does not exist anymore i think, i'll have to check. Please wait before merge.